x-pack/plugins/security_solution/common/endpoint/service/authz/authz.ts

```diff
@@ -179,7 +179,7 @@ export const calculateEndpointAuthz = (
     canKillProcess: canWriteProcessOperations && isEnterpriseLicense,
     canSuspendProcess: canWriteProcessOperations && isEnterpriseLicense,
     canGetRunningProcesses: canWriteProcessOperations && isEnterpriseLicense,
-    canAccessResponseConsole: hasEndpointManagementAccess && isEnterpriseLicense,
```

Comment:
is removal of license gate here intentional?

Reply:
no this is not intentional thanks for the catch